substrate/client/transaction-pool/src/fork_aware_txpool/view_store.rs

```diff
@@ -914,23 +914,23 @@ where
 	/// blocks.
 	pub(crate) fn provides_tags_from_inactive_views(
 		&self,
-		block_hashes: Vec<&HashAndNumber<Block>>,
-		mut xts_hashes: Vec<ExtrinsicHash<ChainApi>>,
+		of_blocks: Vec<&HashAndNumber<Block>>,
+		mut for_xts: Vec<ExtrinsicHash<ChainApi>>,
```

Comment:
Let's keep old naming convention, will be coherent across the pool code base.
```suggestion
		block_hashes: Vec<&HashAndNumber<Block>>,
		mut xts_hashes: Vec<ExtrinsicHash<ChainApi>>,
```